.cfignore

```diff
@@ -9,3 +9,8 @@ cg-deck
 cg-dashboard
 /screenshots/
 static/tests/
+.org.chromium*
+# only exclude these index.html files from the VNC service.
+# they only appear in the root folder.
+/index.html*
```

Comment:
all of them are safe to delete! :) do we want to add a script to package.json? or just tell people to run `rm index.html*`

also, do you know of any tools that work like `watch` but instead of compiling we could delete them? then the user wouldn't have to worry about it at all.